drivers/hmis/lib/form_data/fragments/information_date.json

```diff
@@ -0,0 +1,14 @@
+{
+  "type": "GROUP",
+  "link_id": "information-date",
+  "text": "Information Date",
```

Comment:
Should this be ExitDate?

Reply:
No, this fragment is for the Information Date which is present on the Update and Annual assessments only